### DISCUSSION TITLE
including full DOM access？

### DISCUSSION THREAD

**Discussion by jinxiaoman** [lukehaas/RunJS#621] [Q&A] [answered]
including full DOM access？What does this mean? Can I write HTML? Get DOM node? alert? I didn't find an example, can you provide one?

**Comment by lukehaas** [comment, marked as answer]
It means you have access to the DOM similar to a browser environment.

![Screenshot 2024-05-10 at 16 31 59](https://github.com/lukehaas/RunJS/assets/741344/e84b1749-d3c5-4f45-ba69-256380ef27f6)

You can get a DOM node via the DOM API methods such as `getElementById()`, you can also create elements via `createElement()`.

You could write HTML by doing something like this:
```
document.body.innerHTML = `
  <div id="example">hello, world</div>
`;
```
